Dockerfile

```diff
@@ -1,15 +1,19 @@
-# To build:
-# $ docker run --rm -v $(pwd):/go/src/github.com/micahhausler/k8s-oidc-helper -w /go/src/github.com/micahhausler/k8s-oidc-helper golang:1.7  go build -v -a -tags netgo -installsuffix netgo -ldflags '-w'
-# $ docker build -t micahhausler/k8s-oidc-helper .
-#
-# To run:
-# $ docker run micahhausler/k8s-oidc-helper
-
-FROM busybox
+FROM golang:1.9
```

Comment:
If you are really interested in saving space, you might consider changing this to:
`FROM golang:1.9-alpine3.6`
That image is 83MB instead of 273MB for `golang:1.9`
If you build the binary outside of the container, you can use `scratch` and the resulting container down to between 5 - 10 MB.